tsconfig.json

```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es5",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "es2019",                                     /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
Afaik our newest runners are all ought to be on Node 16+.
According to https://node.green/, `es2021` is fully supported by Node 16. Bump it up?

Reply:
That make sense, will update it !